cs/src/comm/epoxy-transport/Frame.cs

```diff
@@ -13,11 +13,12 @@ namespace Bond.Comm.Epoxy
 
     internal enum FrameletType
     {
-        EpoxyConfig = 0x4743,
-        EpoxyHeaders = 0x5248,
-        LayerData = 0x594C,
-        PayloadData = 0x5444,
-        ProtocolError = 0x5245,
+        EpoxyConfig = 0x4743,           // "GC"
+        EpoxyHeaders = 0x5248,          // "RH"
+        ErrorData = 0x4445,             // "DE"
```

Comment:
They're in alphabetical order now, which, in my experience is the only order for things like this that is maintainable over time, especially when doing three-way merges.